JASP-Desktop/gui/jaspversionchecker.cpp

```diff
@@ -47,6 +56,14 @@ void JASPVersionChecker::downloadFinished()
 	if (latest > cur)
 		emit showDownloadButton(downloadfile);
 
+	if(KnownIssues::issues()->downloadNeededOrLoad())	downloadKnownIssues();
+	else												deleteLater(); //Remove yourself!
+}
```

Comment:
I would set a JASPVersionChecker as member mainWindow and a add a method to start the check. This deleteLater while waiting for a network response is a bit scary... and is not really needed.

Reply:
The deleteLater isn't called while waiting for a network response...